.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: python
-branches:
-    only:
-        - master
-        - dogfood
+
```

Comment:
Forgot that this would automatically push to here.  I just wanted to see the travis errors generated.  I will blast this last commit